auto_nag/scripts/not_landed.py

```diff
@@ -208,70 +168,84 @@ def handle_attachment(self, attachment, res):
     def get_patch_data(self, bugs):
         """Get patch information in bugs
         """
-        nightly_pats = Bugzilla.get_landing_patterns(channels=["nightly"])
+        nightly_pat = Bugzilla.get_landing_patterns(channels=["nightly"])[0][0]
 
         def comment_handler(bug, bugid, data):
-            r = Bugzilla.get_landing_comments(bug["comments"], [], nightly_pats)
-            landed = bool(r)
-            if not landed:
-                for comment in bug["comments"]:
-                    comment = comment["text"].lower()
-                    if "backed out" in comment or "backout" in comment:
-                        landed = True
-                        break
-
-            data[bugid]["landed"] = landed
-
-        def attachment_handler(attachments, bugid, data):
-            res = {}
+            # if the last comment contains a backout: don't nag
```

Comment:
Why only the last comment?

Reply:
My idea was that the bug is kinda stalled at this point.
But if there is activity after the backout then it is no more stalled.
Anyway maybe we shouldn't nag at all when there was a backout, wdyt ?
If the faulty patch has been amended then afaik it has always the "accepted" status so it's hard the guess if the patch is ready to land.

Reply:
Yeah it's probably better to avoid nagging when there was a backout, even if it was in a previous comment.
Maybe we should only rely on data from Phab though